### ISSUE TITLE
Use a default import

### ISSUE THREAD

**Issue by landakram** [landakram/remark-wiki-link#12]
In https://github.com/landakram/remark-wiki-link/pull/6, I removed the default import to prevent a warning in rollup. Almost all of the `remark-*` tools use a default import, so it's a little unexpected for people. It also fails silently.

Change the import to use a default import.

See:

* https://github.com/landakram/remark-wiki-link/issues/8
* https://github.com/landakram/remark-wiki-link/issues/11

**Comment by entityleak** [comment]
I'm using this plugin with a project built on top of Dendron, nuxt and Nuxt Content. It needs a default export to work here.

in `nuxt.config.js`

```
export default {
content: {
  markdown:{
    remarkPlugins: ['remark-wiki-link'],
  }
}
```

An update here to make it integrate seamlessly would be great :)

**Comment by landakram** [comment]
Apologies, I haven't had time to fix this yet. But the problem is that, when I upgraded to support breaking changes in unified, I decided to use rollup to support both cjs and esm module systems and ran into [this issue](https://github.com/rollup/rollup/issues/1961). Essentially, default exports don't work well in esm and so rollup disallows them. So I removed the default export since I wanted to maintain maximum compatibility with different JS environments. Unfortunately, the remark/unified ecosystem is built around default exports, so the decision doesn't make much sense in retrospect.

I think only supporting cjs and providing a default export is a fine solution. If you or anyone else reading are able to make a fix, I would encourage and appreciate that. Otherwise, I will hopefully get to this soon.
